pkgs/development/libraries/phonetisaurus/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv
+, fetchFromGitHub
+, openfst
+, pkg-config
+, python3
+}:
+
+stdenv.mkDerivation rec {
+  pname = "phonetisaurus";
+  version = "2020-07-31";
```

Comment:
Needs to be unstable to be compatible with openfst.

Reply:
I asked for a release here: https://github.com/AdolfVonKleist/Phonetisaurus/issues/61